templates/projects/splash.macros.j2

```diff
@@ -25,27 +33,17 @@
   {% set client = page.config.client[0] if page.config.client else none %}
   {% set contrib = page.config.contributors[0] if page.config.contributors else none %}
 
-  {% set slug = ['/', page.slug[:-6] if page.slug.endswith('/index') else page.slug, '/']|join %}
-  {% set link = slug if (project.owner or project.case_study) else '/work/' %}
-  {% set link = '/open-source/' if (contrib and link == '/work/') else link %}
-
-  {% set images = page.config.image or none %}
+  {% set url = ('/' + page.slug|replace('/index', '') + '/') %}
+  {% set type = '/open-source/' if contrib else '/work/' %}
+  {% set link = url if (project.owner or project.case_study) else type %}
 
-  {% if images %}
-    {% set splash = images|selectattr('type', 'equalto', 'splash')|list or none %}
-    {% set image = splash[0].src if splash else images[0].src %}
-    {% set image = image if '://' in image else ['/static/images/', image]|join('') %}
-
-    <a href="{{ link }}" class="splash-project">
-      <div class="splash-image" style="background-image: url('{{ image }}');"></div>
-
-      <h2 class="splash-client">
-        {{ client.name if client else project.name }}
-      </h2>
-
-      <h3 class="splash-title">
-        {{ project.name if client else [birds.get_bird_name(contrib.author), contrib.role]|join(', ') }}
-      </h3>
+  <article class="splash-project">
+    <a href="{{ link }}" class="splash-logo">
+      {% include 'projects/_%s.svg' % (page.config.logo or 'mozilla') %}
```

Comment:
Is it weird that we use the company logo for Mozilla and SRAM, but the project logo for CoachHub (instead of ORCAS)?

![screenshot 2017-10-30 15 44 07](https://user-images.githubusercontent.com/552316/32191959-2eb9f54c-bd89-11e7-8015-995d9e8cb3a9.jpg)

Reply:
I don't know, but I couldn't find a suitable ORCAS logo. I might be able to replicate it with the right font (proxima nova, maybe?).

Reply:
Yeah, I suppose http://www.orcasinc.com/wp-content/uploads/2017/01/orcas-globalmed-125.png is too small. Could ask them for a logo, or just not worry about it. It's not really a big deal; most people probably won't even notice.